src/containers/AddCheckInOut/index.tsx

```diff
@@ -4,21 +4,25 @@ import FormControlLabel from '@mui/material/FormControlLabel';
 import Switch from '@mui/material/Switch';
 import { useEffect, useState } from 'react';
 import useCheckInOut from '@/hooks/useCheckinout';
+import useCurrentUser from '@/hooks/useCurrentUser';
 
 const AddCheckInOut = () => {
-  const { latestCheckinData } = useCheckInOut();
-
+  const { currentUserData } = useCurrentUser();
+  const role = currentUserData?.data?.data?.role;
+  let isEmployee = false;
+  if (role === 'Employee') {
+    isEmployee = true;
+  }
+  const employee = { isEmployee };
+  const { latestCheckinData, addCheckInAction, updateCheckInAction } =
+    useCheckInOut(employee);
```

Comment:
we might not need to assign employee.
`useCheckInOut({isEmployee})` should work fine.